src/docs/config/overview.md

```diff
@@ -119,17 +125,29 @@ During production builds, the content of each generated file is hashed to repres
 hashFileNames: true
 ```
 
+## invisiblePrehydration
 
-## hashedFileNameLength
+*default: `true`*
 
-*default: `8`*
+When `true`, `invisiblePrehydration` will visually hide components before they are hydrated by adding an automatically injected style tag to the document's head. Setting `invisiblePrehydration` to `false` will not inject the style tag into the head, allowing you to style your web components pre-hydration. 
 
-When the `hashFileNames` config is set to `true`, and it is a production build, the `hashedFileNameLength` config is used to determine how many characters the file name's hash should be.
+> Note: Setting `invisiblePrehydration` to `false` will cause everything to be visible when your page is loaded, causing a more prominent Flash of Unstyled Content (FOUC). However, you can style your web component's fallback content to your preference.
```

Comment:
no action required - tiny diff b/w this and https://github.com/ionic-team/stencil-site/pull/924/files#diff-689c19470ae2ed5ac0b4ba2bb534871dd960e99ae1638ca7da6bab934a5dd022R134

"a" more prominent (emphasis on the article) is missing in #924 - I'm more making the note here for if/when we run into merge conflicts